cmake/CodeCoverage.cmake

```diff
@@ -149,6 +151,14 @@ FUNCTION(SETUP_TARGET_FOR_COVERAGE _targetname _testrunner _outputname)
 		COMMAND ${LCOV_PATH} --remove ${coverage_info} 'tests/*' '/usr/*' --output-file ${coverage_cleaned}
 		COMMAND ${GENHTML_PATH} -o ${_outputname} ${coverage_cleaned}
 
+		# Upload to Codecov (added by @brianhou)
```

Comment:
It would be good to add this change to the [changlog at the top of this file](https://github.com/personalrobotics/aikido/pull/495/files#diff-c647cf0f727c3b16f40757169b38502cL37).